### PR TITLE
Carousels 2016 updates

### PR DESCRIPTION
**Pull request by yatil**

### CODE REVIEW

**Comment by lakeen** [comment]
The 2 paragraphs below are too similar.  Suggest removing the first one.

Carousels allow users to select to view one item from a series of content items. Common examples are slide shows other sliders, like scrolling news headlines, images or featured articles. Carousels often move from item to item automatically, which can be too fast to read or distracting for some users.

Why is this important?

Carousels, also referred to as “slide shows” and “sliders”, display one item from a series of content items. Common examples are scrolling news headlines and images, like featured articles. Carousels often use animations to move from item to item, which can be distracting for some users. They may also move too fast for some users to be able to read.

**Comment by lakeen** [comment]
"All functionalit needs to be usable"  functionality is misspelled.

**Comment by bakkenb** [comment]
I agree with Laura. Those paragraphs are very similar and should be combined or the best one selected and the other removed. 

I don't understand the "Why this is important?" title either. I guess if you removed the complete example sentence and the extra paragraph, then the question title would make more sense by answering the first statement "Develop carousels in an accessible way..."

**Comment by bakkenb** [comment]
I like the new structure with the Complete Example in the menu navigation at the bottom of the list. Structure wise, I think it is in good shape. I understand that you are still working on content.

**Comment by yatil** [comment]
@lakeen & @bakkenb The paragraph shouldn’t appear two times… Sorry for the confusion. I’ll check that!

File: source/carousels/structure.html.erb.md [hunk 1]
```diff
@@ -33,12 +35,12 @@ See the Page Structure Tutorial for more information on [regions](/page-structur
 {:/nomarkdown}
 
 ~~~html
-<section class="carousel" aria-labelledby="carouselheading">
+<section … aria-labelledby="carouselheading">
```
Comment (yatil) [inline source/carousels/structure.html.erb.md]: Can we keep the carousel classes? Otherwise it gets very, very general.

**Comment by yatil** [comment]
@bakkenb:

> I don't understand the "Why this is important?" title either. I guess if you removed the complete example sentence and the extra paragraph, then the question title would make more sense by answering the first statement "Develop carousels in an accessible way..."

“Why is this important?” is basically our section where we talk about specific benefits for people with disabilities (and advantages in other situations). [I wonder if we can tie that in to the Perspectives videos (”essential for some, useful for all”) and HPWDUTW (How People with Disabilities Use The Web)  at some time.]

**Pull request merged**